hypothesisfuzzer/fuzz_server.py

```diff
@@ -1,24 +1,31 @@
 import json
 import logging
+import logging.handlers
 import os
 import yaml
 
+from .errors import ConfigMissingOptionException
 from .fuzzing import RepoFuzzer
 from flask_sqlalchemy import SQLAlchemy
 from flask_cors import CORS
 from flask import Flask, request, send_from_directory
-from .errors import ConfigMissingOptionException
+from sys import platform
+
+
+logging.basicConfig(datefmt='%d-%b-%y %H:%M:%S',
+                    filename='fuzz_server.log',
+                    format='%(asctime)s - %(levelname)s - %(message)s',
+                    level=logging.DEBUG)
+
+if platform.startswith('linux'):
+    handler = logging.handlers.SysLogHandler(address='/dev/log')
+    logging.getLogger('logger').addHandler(handler)
 
 
 class FuzzServer:
 
     def __init__(self, config_path='config.yml'):
 
-        logging.basicConfig(datefmt='%d-%b-%y %H:%M:%S',
-                            filename='fuzz_server.log',
-                            format='%(asctime)s - %(levelname)s - %(message)s',
-                            level=logging.DEBUG)
-
         self.log = logging.getLogger('logger')
```

Comment:
I mean get rid of self.log entirely.
Make it a global instead, e.g. on line 19 have `logger = logging.getLogger('logger')` since all the loggers are the same instances anyways, theres no point encapsulating it in a class.

Also can you change it to `logging.getLogger(__name__)`? That will make each logger specific to the file/class name.